pkg/pillar/types/domainmgrtypes.go

```diff
@@ -232,15 +232,16 @@ func (config DomainConfig) LogKey() string {
 // so-called "fixed resources", which means that the virtual machine
 // must be restarted before changes to the field will take effect.
 type VmConfig struct {
-	Kernel     string // default ""
-	Ramdisk    string // default ""
-	Memory     int    // in kbytes; Rounded up to Mbytes for xen
-	MaxMem     int    // in kbytes; Default equal to 'Memory', so no ballooning for xen
-	VCpus      int    // default 1
-	MaxCpus    int    // default VCpus
-	RootDev    string // default "/dev/xvda1"
-	ExtraArgs  string // added to bootargs
-	BootLoader string // default ""
+	Kernel                 string // default ""
+	Ramdisk                string // default ""
+	Memory                 int    // in kbytes; Rounded up to Mbytes for xen
+	MaxMem                 int    // in kbytes; Default equal to 'Memory', so no ballooning for xen
+	VCpus                  int    // default 1
+	MaxCpus                int    // default VCpus
+	RootDev                string // default "/dev/xvda1"
+	ExtraArgs              string // added to bootargs
+	BootLoader             string // default ""
```

Comment:
Confusing name IMO and UEFI is not a bootloader

Reply:
It was already this way. I just added whitespeaces.